theme/sign-in-off/_base.scss

```diff
@@ -1,7 +0,0 @@
-#wb-so {
```

Comment:
Well, that is a funny, one, the only way to make it see that it was a move is to have one commit for the move, and one for the edit. What do you prefer, fewer commit or tracking the file move?